x-pack/plugins/index_management/public/application/sections/home/data_stream_list/edit_data_retention_modal/edit_data_retention_modal.tsx

```diff
@@ -146,13 +151,94 @@ const configurationFormSchema: FormSchema = {
       }
     ),
   },
+  dataRetentionEnabled: {
+    type: FIELD_TYPES.TOGGLE,
+    defaultValue: false,
+    label: i18n.translate(
+      'xpack.idxMgmt.dataStreamsDetailsPanel.editDataRetentionModal.dataRetentionEnabledField',
+      {
+        defaultMessage: 'Enable data retention',
```

Comment:
```suggestion
        defaultMessage: 'Enable data retention',
```
What does this do when it's disabled? No data is kept? Or there's still a minimum?
Wondering:
- if it's useful to show a little help text under the toggle to be explicit about what happens to the data when it's disabled (it might be obvious but I don't know this part very well ^^')
- What's the "value" we show in the flyout and table under Data retention when it's disabled? Does it hint on what happens to the data (if useful)? Sorry I tried to test it locally but my Kibana shows a blank screen after the loading screen.

Reply:
When its disabled we show in the UI just a text saying `Disabled`, and when its infinite data retention we show a value of `∞`. I'll ask around whats the default behaviour when its disabled, as I'm not 100% sure

Reply:
I'll go ahead and merge this work in, will follow up with you once i get an answer!